synfig-studio/src/gui/resources/CMakeLists.txt

```diff
@@ -1 +1,17 @@
 add_subdirectory(css)
+set(INSTALL_DESTINATION "share/synfig/resources")
+set(RESOURCES_FILES
+	canvas_options.glade
+	preview_options.glade
+	dialog_workspaces.glade
+	shortcuts_window.glade
+)
+
+file(COPY ${RESOURCES_FILES}
+	DESTINATION ${SYNFIG_BUILD_ROOT}/${INSTALL_DESTINATION}
+)
```

Comment:
Use css style `configure_file(... COPYONLY)` instead of `file(COPY ...)`

https://github.com/synfig/synfig/blob/520d9721a9d28243f8d2ba2efce4c4178a8ecf9a/synfig-studio/src/gui/resources/css/CMakeLists.txt#L7-L9

This makes sure when the user changes any glade file the output directory is updated with the new file when we run `make`
Whereas, `file(COPY ...)` works only if the user executes `cmake` first and then `make` which is quite unrealistic because IDE only cares about `make` command after the project is set up. In other words, the file is never updated to our build directory. And we're stuck with the old glade file.

Reply:
Resolved :)